state/StateGetPointerToData.F90

```diff
@@ -0,0 +1,78 @@
+#include "MAPL_ErrLog.h"
+
+module mapl3g_StateGetPointerToData
+
+   use mapl_ErrorHandling
+   use mapl_KeywordEnforcer
+   use esmf
+
+   implicit none
+   private
+
+   public :: StateGetPointerToData
+
+   interface StateGetPointerToData
+      module procedure StateGetPointerToDataR4_1
+      module procedure StateGetPointerToDataR4_2
+      module procedure StateGetPointerToDataR4_3
+      module procedure StateGetPointerToDataR4_4
+      module procedure StateGetPointerToDataR8_1
+      module procedure StateGetPointerToDataR8_2
+      module procedure StateGetPointerToDataR8_3
+      module procedure StateGetPointerToDataR8_4
+   end interface StateGetPointerToData
+
+contains
+
+#define NAME_ StateGetPointerToData
+
+#define TYPEKIND_ R4
+
+! StateGetPointerToDataR4_1
+#define RANK_ 1
```

Comment:
Glad you are using macros.   Should make it easy to revise names here.

Reply:
I didn't really do much there - that's how it is in MAPL2.

Reply:
ok - kudos retracted.

Reply:
:-(